code/modules/reagents/machinery/reagentgrinder.dm

```diff
@@ -399,55 +402,43 @@ var/global/list/juice_items = list (
 		holdingitems -= O
 	holdingitems = list()
 
-/obj/machinery/reagentgrinder/proc/is_allowed(var/obj/item/weapon/reagent_containers/O)
+/proc/is_allowed(var/obj/item/weapon/reagent_containers/O)
 	for (var/i in blend_items)
 		if(istype(O, i))
 			return 1
 	return 0
 
-/obj/machinery/reagentgrinder/proc/get_allowed_by_id(var/obj/item/weapon/grown/O)
+/proc/get_allowed_by_id(var/obj/item/weapon/grown/O)
```

Comment:
Same as above

Reply:
reorganised it so the argument is now the thing the proc is called on, if thats any better. mortars use this too so that's my reasoning